helm/kubernetes-cluster-prep/templates/tests/_helm_test.bats

```diff
@@ -0,0 +1,33 @@
+# Define a Helm named template so that this BATS script can be included
+# in a Helm test ConfigMap. This can be left commented out; Helm will
+# still be able to find this template, and this file can be kept as
+# valid BATS syntax.
+#
+#{{- define "kube-cluster-prep.helm-test.bats" }}
+#!/usr/bin/env bats
+
+# Run out of same directory in which this script resides
+cd "/tests"
+source "./utils.sh"
+
+source "/bats/bats-support/load.bash"
+source "/bats/bats-assert/load.bash"
+source "/bats/bats-file/load.bash"
+
+# Baseline BATS test result color
+text_color "$MAGENTA"
+
+@test "Conjur Appliance URL is a reachable address" {
+  status=0
```

Comment:
Now that we're loading `bats-assert`, we can use that functionality to condense this test case a little:
```
@test "Conjur Appliance URL is a reachable address" {
  cmd=(curl -k --connect-timeout 5 "$conjurApplianceUrl":443)
  display_info "Running ${cmd[@]}"
  run "${cmd[@]}"
  if [ "$status" -ne 0 ]; then
    display_error "Please check configured Conjur Appliance URL.\n" \
                  "It is not reachable via 'curl -k'"
  fi
  assert_success
}

Reply:
ok, added it.
(I had to add an assert_failure after the display_error.)